das_client/analysis_options.yaml

```diff
@@ -13,16 +13,27 @@ linter:
   # The lint rules applied to this project can be customized in the
```

Comment:
Not for this PR but would be nice if we could take some of the Flutter style guidelines to linter like [ordering of class members](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#order-other-class-members-in-a-way-that-makes-sense).